App/Sagas/TextileEventsSagas.ts

```diff
@@ -88,25 +86,6 @@ export function * ignoreFileRequest() {
   }
 }
 
-export function * appStateChange() {
-  while (true) {
-    try {
-      // Block until we get an active or background app state
-      const action: ActionType<typeof TextileEventsActions.appStateChange> =
-        yield take((action: RootAction) =>
-          action.type === getType(TextileEventsActions.appStateChange)
-        )
-
-      if (yield select(PreferencesSelectors.showNodeStateNotification)) {
-        yield call(displayNotification, 'App State Change: ' + action.payload.newState)
-      }
-
-      yield call(logNewEvent, 'State Change', action.payload.newState)
-    } catch (error) {
-      yield call(logNewEvent, 'appStateChange', error.message, true)
-    }
-  }
-}
```

Comment:
"App state" is no longer exposes by the sdk (it's used internal to the native sdks, but that is an implementation detail that could change in the future). It's irrelevant to an end user app, and it's something the developer can access themselves using existing platform apis.